tests/15-EdvClientRevokeCapability.spec.js

```diff
@@ -1,13 +1,17 @@
 /*!
  * Copyright (c) 2018-2020 Digital Bazaar, Inc. All rights reserved.
  */
-import jsigs from 'jsonld-signatures';
-import uuid from 'uuid-random';
+import {CapabilityDelegation} from '@digitalbazaar/zcapld';
+import {createRecipient, JWE_ALG} from './test-utils.js';
 import {EdvClient} from '..';
+import {Ed25519Signature2020} from '@digitalbazaar/ed25519-signature-2020';
+import jsigs from 'jsonld-signatures';
 import mock from './mock.js';
-const {SECURITY_CONTEXT_V2_URL, sign, suites: {Ed25519Signature2018}} = jsigs;
-import {CapabilityDelegation} from 'ocapld';
-import {createRecipient, JWE_ALG} from './test-utils.js';
+import uuid from 'uuid-random';
+import {constants} from 'ed25519-signature-2020-context';
```

Comment:
This is correct. However, the latest `Ed25519Signature2020`does export the context as well.  If this export is used, this can be simplified and the `ed25519-signature-2020-context` dependency can likely be removed.  It would be good to update the pattern used here for demonstration purposes.

https://github.com/digitalbazaar/ed25519-signature-2020/blob/main/lib/Ed25519Signature2020.js#L198-L199

Reply:
Updated 082e3e9  & e57ddd9